apm-agent-plugins/apm-process-plugin/src/main/java/co/elastic/apm/agent/process/ProcessHelper.java

```diff
@@ -72,8 +72,6 @@ void doStartProcess(@Nonnull AbstractSpan<?> parentContext, @Nonnull Process pro
 
         Span span = parentContext.createSpan()
             .withType("process")
-            .withSubtype(binaryName)
-            .withAction("execute")
```

Comment:
Why? Risk of too high cardinality?

Reply:
Yes, while the total number of executables would still be quite limited in practice, there is no extra value to have this as sub-type as everything is already included in the name. Also, there would be no other action but `execute`.